tests/unit/test_mixins.py

```diff
@@ -34,6 +34,7 @@ def _make_user_mock(user_id):
 
 
 class CommonMixinGuineaPig(object):
+
```

Comment:
Well technically a cause not a reason. Cause is that I was trying to add ``@XBlock.wants('settings')`` to ``SettingsMixin`` which had couple of bad side efects. In the end i decided not to, and only trace left are these lines :). I'll remove them of course.